registration/templates/registration/dealer/dealer-locate.html

```diff
@@ -38,14 +38,14 @@ <h1>Dealer Registration - {{event}}</h1>
                 <div class="form-group">
                     <label for="email" class="col-sm-3 control-label">Email</label>
                     <div class="col-sm-9">
-                        <input type="email" id="email" placeholder="Email" class="form-control" required data-error="Email is required."/>
+                        <input type="email" name="email" id="email" placeholder="Email" class="form-control" required data-error="Email is required."/>
```

Comment:
Still need to sort out populating this from get parameter?  I don't see how dealers could have actually ever worked.